pkg/authhelpers/gcpauth.go

```diff
@@ -39,6 +40,15 @@ func ResolveGoogleCredentials(ctx context.Context, scope string) (*google.Creden
 		return credentials, nil
 	}
 
+	if creds := os.Getenv("GOOGLE_OAUTH_ACCESS_TOKEN"); creds != "" {
+		// We try $GOOGLE_OAUTH_ACCESS_TOKEN before gcp.DefaultCredentials
```

Comment:
Probably want a comment similar to the above about GOOGLE_CREDENTIALS about this not being a google standard, but another thing we're copying from how the gcp provider is setup.

Reply:
Fair point. Comment added.